tests/network/hotplug.go

```diff
@@ -238,6 +238,28 @@ var _ = SIGDescribe("nic-hotplug", func() {
 			Entry("In place", decorators.InPlaceHotplugNICs, inPlace),
 			Entry("Migration based", decorators.MigrationBasedHotplugNICs, migrationBased),
 		)
+
+		DescribeTable("VMI status should report the MAC address specified in the VM template spec", func(plugMethod hotplugMethod) {
```

Comment:
I'm a bit confused here.

- This seems very similar to the test on line 100 which hotplug an interface and checks to see it is there in the guest.
  - The addition here is to see the mac address on the interface status, is this correct?
  - If it is correct, why not just add the mac check to the original test?